src/lib/renderer/renderer.tsx

```diff
@@ -212,13 +241,33 @@ export const RenderComponent = ({
 const Fallback = (props: IComponentProps) => <React.Fragment>{props.children}</React.Fragment>;
 
 /**
- * Custom hook resolving the `React.ComponentType` from `RenderContext`.
+ * Custom hook resolving the `React.ComponentType` based on the configuration in `RenderContext`.
+ * Resolving is performed in the following order:
+ *
+ *   1. - A `React.ComponentType` configured for a certain "containers" entry indicating an internal
+ *        edge case.
+ *   2. - A `React.ComponentType` configured for a certain "components" entry indicating a regular
+ *        component.
+ *   3. - A fallback component solely rendering `props.children`.
  * @external {RenderContext} Expects `RenderContext` to be available.
  */
 export const useComponentType = (
   component: IRenderable
 ): React.ComponentType<IColumnProps | IComponentProps> => {
   const renderConfiguration = useContext(RenderContext);
+  const ContainerType = useContainerConfiguration(component);
   const ComponentType = renderConfiguration.components[component.type];
-  return ComponentType || Fallback;
+  return ContainerType || ComponentType || Fallback;
+};
+
+/**
+ * Returns the applicable `containerConfiguration` (if any) for component.
+ * @see {IContainerConfiguration}
+ */
+export const useContainerConfiguration = (component: IRenderable) => {
+  const renderConfiguration = useContext(RenderContext);
+  if (component.multiple) {
+    return renderConfiguration.containers.multiple || null;
```

Comment:
`|| null` is voor de cases dat de mapping mist? Moeten we dan niet terugvallen naar de default mapping?

Reply:
That would mean that this behavior can't be turned off. For instance, if a design system would have a multiple capable input, it would be overridden by our Multiple container. Passing a falsy value can turn it off and allow for the textfield to be used instead.